jobs/ml_jobs/embeddings/utils/file_handler.py

```diff
@@ -37,6 +26,28 @@ def _load_images(batch_urls):
                     "status": "failed",
                 }
             )
+        else:
```

Comment:
peux tu rajouter un Commentaire 
`TODO: refactor this to remove the try if possible, otherwise an Exception more specialized`